tests/functional/test_install_upgrade.py

```diff
@@ -362,7 +362,7 @@ def test_from_distribute_6_to_setuptools_7(
         )
         result = self.script.run(
             self.ve_bin / 'pip', 'list',
-            expect_stderr=True if sys.version_info[:2] == (2, 6) else False,
+            expect_stderr=True,
```

Comment:
And again. Why don't we expect stderr for versions other than 2.6, and is that because of this change?